src/mon/OSDMonitor.cc

```diff
@@ -7652,28 +7755,28 @@ bool OSDMonitor::prepare_command_impl(MonOpRequestRef op,
     }
     if (prefix == "osd crush weight-set create") {
       if (osdmap.require_min_compat_client > 0 &&
-	  osdmap.require_min_compat_client < CEPH_RELEASE_LUMINOUS) {
-	ss << "require_min_compat_client "
-	   << ceph_release_name(osdmap.require_min_compat_client)
-	   << " < luminous, which is required for per-pool weight-sets. "
+	      osdmap.require_min_compat_client < CEPH_RELEASE_LUMINOUS) {
+	    ss << "require_min_compat_client "
+	       << ceph_release_name(osdmap.require_min_compat_client)
+	       << " < luminous, which is required for per-pool weight-sets. "
            << "Try 'ceph osd set-require-min-compat-client luminous' "
            << "before using the new interface";
-	err = -EPERM;
-	goto reply;
+	    err = -EPERM;
+	    goto reply;
       }
       string poolname, mode;
       cmd_getval(cct, cmdmap, "pool", poolname);
       pool = osdmap.lookup_pg_pool_name(poolname.c_str());
       if (pool < 0) {
-	ss << "pool '" << poolname << "' not found";
-	err = -ENOENT;
-	goto reply;
+	    ss << "pool '" << poolname << "' not found";
+	    err = -ENOENT;
+	    goto reply;
       }
       cmd_getval(cct, cmdmap, "mode", mode);
       if (mode != "flat" && mode != "positional") {
-	ss << "unrecognized weight-set mode '" << mode << "'";
-	err = -EINVAL;
-	goto reply;
+	    ss << "unrecognized weight-set mode '" << mode << "'";
+	    err = -EINVAL;
+	    goto reply;
```

Comment:
same whitespace nit here

Reply:
These look to now be "properly" intended in the source file itself...